Gopkg.toml

```diff
@@ -181,16 +181,20 @@ ignored = ["github.com/Sirupsen/logrus", "github.com/gravitational/license/gener
   name = "github.com/gravitational/license"
   version = "0.0.6"
 
+[[constraint]]
+  name = "gopkg.in/yaml.v2"
+  version = "v2.2.8"
+
 [[override]]
   name = "k8s.io/api"
-  branch = "release-1.9"
+  version = "v0.17.3"
 
 [[override]]
   name = "k8s.io/apimachinery"
-  branch = "release-1.9"
+  version = "v0.17.3"
 
 [[constraint]]
-  version = "6.0.0"
+  version = "v0.17.3"
   name = "k8s.io/client-go"
```

Comment:
For anyone as thrown off by this diff as I was, it appears that the versioning strategy was [changed](https://github.com/kubernetes/client-go/blob/7624422635d15c33463f6a85ac2ccb1b0db452ec/INSTALL.md) at some point, and this is indeed an upgrade.